tests/simple/verifySignature31.uplc.expected

```diff
@@ -0,0 +1 @@
+(con bool True)
```

Comment:
I don't see the file `tests/simple/verifySignature31.uplc` here, should that be committed too? Otherwise these tests won't run on CI.